### PR TITLE
tfs-391073 createFromBase64EncodedAccessKey

### PR DESCRIPTION
**Pull request by paolo-arge-lf**
- Remove class AccessKeyImpl since not needed
- export createFromBase64EncodedAccessKey from module
- createFromBase64EncodedAccessKey: remove try catch so to not obfuscate exceptions

https://tfs.laserfiche.com/workitems?id=391073

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@laserfiche/lf-api-client-core",
-  "version": "1.0.9",
+  "version": "1.0.10",
```
Comment (sxu1998) [inline package.json:3]: may you do an npm i so the package-lock.json file also gets updated with 1.0.10?

File: lib/OAuth/AccessKey.test.ts [hunk 2]
```diff
@@ -1,54 +1,56 @@
...
-    }
-  );
+    //Act
+
+    const decodedAccessKey: AccessKey = createFromBase64EncodedAccessKey(base64EncodedAccessKey);
+
+    //Assert
+    const accessKeyJSON: string = JSON.stringify(expectedAccessKey);
+    const decodedAccessKeyJSON: string = JSON.stringify(decodedAccessKey);
+    console.log(decodedAccessKeyJSON);
```
Comment (sxu1998) [inline lib/OAuth/AccessKey.test.ts:32]: should we get rid of the console.log statement?

Comment (paolo-arge-lf) [reply lib/OAuth/AccessKey.test.ts]: done

File: package.json (hunk 1 above)
Comment (paolo-arge-lf) [reply package.json:3]: done

**Review by sxu1998 — APPROVED** [review]

**Pull request merged**